spec/models/vacancy_spec.rb

```diff
@@ -499,6 +500,14 @@
     end
   end
 
+  describe 'when supporting documents are provided' do
+    it 'should return the document name' do
+      document = create(:document, name: 'Test.txt')
```

Comment:
We could improve this test by having two different documents attached to a vacancy and checking the names for both. Any thoughts?

Reply:
Yes definitely